user-guide/Standard_Apps/PA/Getting_Started_with_PA/Deploying_and_Configuring_the_PA_Framework.md

```diff
@@ -0,0 +1,103 @@
+---
+uid: Deploying_and_Configuring_the_PA_Framework
+---
+
+# Deploying and configuring the PA framework
+
+## Checking and installing the prerequisites
+
+Before you move on to the PA development and deployment, make sure your DataMiner platform has the necessary resources and specifications for a smooth execution of SRM workflows, as SRM is an essential prerequisite for the PA framework. You can find the expected requirements on the [DataMiner Compute Requirements page](https://community.dataminer.services/dataminer-compute-requirements/).
+
+Your DataMiner System will also need to use an [Elasticsearch database](xref:Elasticsearch_database).
+
+> [!TIP]
+> For more information on how to install Elasticsearch on your DMA, click [here](xref:Installing_Elasticsearch_via_DataMiner).
+
+## Enabling DOM
+
+To make sure DOM data are available in dashboards and applications, make sure the [DOMManager soft-launch option](xref:SoftLaunchOptions#dommanager) is enabled.
```

Comment:
"To make sure DOM data are available in dashboards and applications,"  : it is not for that purpose that we need DOM.